elements/lisk-transaction-pool/test/unit/transaction_pool.spec.ts

```diff
@@ -237,4 +237,66 @@ describe('TransactionList class', () => {
 			expect(status).toEqual(false);
 		});
 	});
+
+	describe('removeTransaction', () => {
+		let txGetBytesStub: any;
+		const tx = {
+			id: '1',
+			nonce: BigInt(1),
+			minFee: BigInt(10),
+			fee: BigInt(1000),
+			senderPublicKey: generateRandomPublicKeys()[0],
+		} as Transaction;
+
+		txGetBytesStub = jest.fn();
+		tx.getBytes = txGetBytesStub.mockReturnValue(Buffer.from(new Array(10)));
+
+		it('should return false when a tx id does not exist', async () => {
+			const addStatus = await transactionPool.addTransaction(tx);
+			expect(addStatus).toEqual(true);
+			expect(transactionPool.getAllTransactions().length).toEqual(1);
+			expect(
+				transactionPool['_transactionList'][
+					getAddressFromPublicKey(tx.senderPublicKey)
+				].get(tx.nonce),
+			).toEqual(tx);
+			expect(transactionPool['_feePriorityQueue'].values).toContain(tx.id);
```

Comment:
Maybe wrap this in beforeEach hook as you are trying to add trx before removal test